src/App.jsx

```diff
@@ -1,33 +1,73 @@
-import { useState } from 'react'
-import reactLogo from './assets/react.svg'
-import viteLogo from '/vite.svg'
-import './App.css'
+import { useState, useEffect } from "react"
+import "./App.css"
+import Inbox from "./components/Inbox"
+import InboxDetails from "./components/InboxDetails"
+import NavBar from "./components/NavBar"
+import Search from "./components/Search"
 
 function App() {
-  const [count, setCount] = useState(0)
+  const [emailData, setEmailData] = useState([])
+
+  useEffect(() => {
+    fetch("https://email-client-api.dev.io-academy.uk/emails")
+      .then((response) => response.json())
+      .then((data) => {
+        setEmailData(data.data)
+      })
+  }, [])
+
+  const [hiddenToggle, setHiddenToggle] = useState("hidden")
+
+  function handleClick() {
```

Comment:
What kind of click is being handled? Is there a more specific name for this function?